.github/workflows/format-repo.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         run: npm install
 
       - name: 👔 Format
-        run: npm run format:repo
+        run: npm run format
```

Comment:
We only have one config now so no longer need separate `format`/`format:repo` and `lint`/`lint:repo` options